src/assets/stylesheets/application/ballots/_about.scss

```diff
@@ -22,6 +22,9 @@
       padding-right: $cell-right-padding;
     }
     &_name {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
```

Comment:
@gabitoesmiapodo Let's return the word wrapping for the `name` (`proposer`) column. The previous look was better.